flang/test/Lower/altalt.f90

```diff
@@ -0,0 +1,19 @@
+! RUN: bbc -emit-fir -o - %s | FileCheck %s
+
+   ! CHECK-LABEL: func @_QPss
+   subroutine ss(n)
+     print*, n
+     ! CHECK: return{{$}}
+     return
+   ! CHECK-LABEL: func @_QPee
+   entry ee(n,*)
+     ! CHECK: return %{{.}} : index
+     return 1
+   end
+
+   ! CHECK-LABEL: func @_QQmain
+     call ss(7)
+     call ee(2, *11)
+     print*, 'default'
+11   print*, 11
+   end
```

Comment:
Looks like this test could be merged with the entry.f90 file.

Reply:
This test has both alternate entries and alternate returns.  I'll merge it in with alternate return test altret.f90 in a way that also addresses the next suggestion.